tests/common/defender/aws_iot_tests_defender_api.c

```diff
@@ -546,48 +631,29 @@ static void _waitForMetricsAccepted( uint32_t timeoutSec )
     uint32_t maxIterations = timeoutSec / _WAIT_STATE_INTERVAL_SECONDS;
     uint32_t iter = 1;
 
-    /* Wait for an valid event type to be set. */
-    while( _callbackInfo.eventType != AWS_IOT_DEFENDER_METRICS_ACCEPTED )
+    /* Wait for an event type to be set. */
+    while( _callbackInfo.eventType == -1 )
```

Comment:
Is there an ENUM value for -1?  better than a magic number...

Reply:
No, it is a magic number. I am not sure what is the best thing to do here. we can deal with it later